From bd5d7422ba4f67bd0870be35e74e49c3efd9109b Mon Sep 17 00:00:00 2001
From: 11111-dongfengjun <dongfengjun2024@163.com>
Date: Thu, 18 Sep 2025 22:13:08 +0800
Subject: [PATCH 72/73] CI 3.1.8 uart

---
 perip/uart16550/rtl/uart_tfifo.v | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/perip/uart16550/rtl/uart_tfifo.v b/perip/uart16550/rtl/uart_tfifo.v
index 71329116..b788bc2d 100644
--- a/perip/uart16550/rtl/uart_tfifo.v
+++ b/perip/uart16550/rtl/uart_tfifo.v
@@ -211,7 +211,7 @@ begin
             begin
                 top   <= #1 top_plus_1;
                 count <= #1 count + 1'b1;
-                $write("%c", data_in);
+                //$write("%c", data_in);
             end
         2'b01 : if(count>0)
             begin
@@ -221,7 +221,7 @@ begin
         2'b11 : begin
                 bottom <= #1 bottom + 1'b1;
                 top    <= #1 top_plus_1;
-                $write("%c", data_in);
+                //$write("%c", data_in);
                 end
         default: ;
         endcase
-- 
2.43.0

