From ab9489576dfded9098d67f1f79b6de676d4c1054 Mon Sep 17 00:00:00 2001
From: 11111-dongfengjun <dongfengjun2024@163.com>
Date: Wed, 17 Sep 2025 19:27:40 +0800
Subject: [PATCH 2/2] CI 2.7.4

---
 bsp/abstract-machine/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/bsp/abstract-machine/Makefile b/bsp/abstract-machine/Makefile
index 8d70f310b..b45a171b0 100644
--- a/bsp/abstract-machine/Makefile
+++ b/bsp/abstract-machine/Makefile
@@ -8,7 +8,9 @@ NAME = rtthread
 SRCS = $(shell find src -name "*.c")
 CFLAGS += -DHAVE_CCONFIG_H -D__RTTHREAD__
 CFLAGS += -Wno-nonnull-compare
-LDFLAGS += -T extra.ld
+ifneq ($(ARCH),riscv32e-ysyxsoc)
+	LDFLAGS += -T extra.ld
+endif
 -include $(FILE_MK)
 -include $(AM_APPS_MK)
 include $(AM_HOME)/Makefile
-- 
2.43.0

